SoftLayer/CLI/block/order.py

```diff
@@ -48,40 +48,48 @@
               help='Optional parameter for ordering snapshot '
               'space along with endurance block storage; specifies '
               'the size (in GB) of snapshot space to order')
+@click.option('--service-offering',
+              help='The service offering package to use for placing '
+              'the order [optional, default is \'storage_as_a_service\'',
+              type=click.Choice([
+                  'storage_as_a_service',
+                  'enterprise',
```

Comment:
I chose "enterprise" to correspond to the name of the SoftLayer_Container_Product_Order_Network_Storage_Enterprise order container (http://developer.softlayer.com/reference/datatypes/SoftLayer_Container_Product_Order_Network_Storage_Enterprise). Calling it "endurance" instead "enterprise" might be more clear within the SLCLI, though. Should I go ahead and update this as well @allmightyspiff ?